Library/Formula/saltstack.rb

```diff
@@ -141,5 +150,22 @@ def caveats; <<-EOS.undent
     system "python", "-c", "import M2Crypto"
 
     system "#{bin}/salt", "--version"
+
+    # Requires write access to not be blocked by '--sandbox'
```

Comment:
That's on by default so: it will be blocked so a test that assumes it won't be will be broken.